google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -154,7 +155,7 @@ public void run() {
                 DebugTarget targetSelection = null;
 
                 if (debuggees == null || debuggees.getDebuggees() == null || debuggees.getDebuggees().isEmpty()) {
-                  disableTargetSelector();
+                  disableTargetSelector(new NoDebuggableModuleException());
```

Comment:
@elharo what do you think of this?  It feels kind of wrong to me to create an exception type just to be able to reuse a throwable based API.